src/components/config_profile/test/CMakeLists.txt

```diff
@@ -42,10 +42,17 @@ set(LIBRARIES
 
 collect_sources(SOURCES "${CMAKE_CURRENT_SOURCE_DIR}")
 create_test(config_profile_test "${SOURCES}" "${LIBRARIES}")
- 
+
 file(COPY smartDeviceLink.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY smartDeviceLink_test.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY smartDeviceLink_invalid_pairs.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY smartDeviceLink_invalid_string.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-file(COPY smartDeviceLink_invalid_int.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR}) 
-file(COPY smartDeviceLink_invalid_boolean.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR}) 
+file(COPY smartDeviceLink_invalid_int.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY smartDeviceLink_invalid_boolean.ini DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+file(COPY smartDeviceLink.ini DESTINATION ${TEST_BIN_DIR})
+file(COPY smartDeviceLink_test.ini DESTINATION ${TEST_BIN_DIR})
+file(COPY smartDeviceLink_invalid_pairs.ini DESTINATION ${TEST_BIN_DIR})
+file(COPY smartDeviceLink_invalid_string.ini DESTINATION ${TEST_BIN_DIR})
+file(COPY smartDeviceLink_invalid_int.ini DESTINATION ${TEST_BIN_DIR})
+file(COPY smartDeviceLink_invalid_boolean.ini DESTINATION ${TEST_BIN_DIR})
```

Comment:
@BSolonenko please add an empty line in the end of file

Reply:
@AKalinich-Luxoft fixed in https://github.com/Ford-Luxoft/sdl_core/pull/21/commits/72faa1e1b1feaf2c018815063d6e5fe771cfe13e